authentication/models.py

```diff
@@ -116,3 +116,10 @@ def get_short_name(self):
         the user's real name, we return their username instead.
         """
         return self.username
+
+    def auth_token(self):
+            self.encoded_jwt = jwt.encode(
+                {'username': self.username,
+                'exp': datetime.now() + timedelta(hours=24)},
+            settings.SECRET_KEY, algorithm='HS256').decode('utf-8')
+            return self.encoded_jwt 
```

Comment:
Invalid indent code.

Reply:
Noted well will rectify this quickly

Reply:
This was rectified i made some changes to this code and indented